### ISSUE TITLE
Error when no replacement suggested

### ISSUE THREAD

**Issue by fehrnah** [rhysd/vim-grammarous#56]
Using latest Grammarous version
Languagetool version: 3.9
Language: French

When there is no replacement for a spelling error, Grammarous uses 'replacement' as the replacement string.
I found the bug on french text, but I think it may be present in other languages

**Comment by rhysd** [comment]
Thank you for reporting. Because I haven’t seen the error which does not have a replacement, I want to start from reproducing. Could you provide a sentence which can reproduce this issue?

**Comment by fehrnah** [comment]
Any french sentence with a spelling mistake triggers the error.

For example:
> J'ai recemment dormis.

It looks like Languagetool disabled spelling suggestion due to it's slowness:
https://forum.languagetool.org/t/no-suggestions-for-typos-in-french-dictionary/732

I don't know if it is possible to enable it or what speed difference it would make.

**Comment by rhysd** [comment]
Thank you. I'll try out.

**Comment by rhysd** [comment]
I tried the text, but I could not see any problem:

<img width="598" alt="2018-01-11 16 02 23" src="https://user-images.githubusercontent.com/823277/34812392-d05abcc2-f6e8-11e7-9892-7098c90f04d5.png">

could you provide the output from following command?:

```sh
$ echo "J'ai recemment dormis." | java -jar /path/to/vim-grammarous/misc/LanguageTool-3.8/languagetool-commandline.jar -c utf-8 -d 'WHITESPACE_RULE,EN_QUOTES' -l fr --api
```

Please ensure that your local vim-grammarous is the latest at first.

And could you give me more information about your config?

- LanguageTool's version downloaded in `/path/to/vim-gramamrous/misc`. If it's older than 3.8, please update it at first by `:GrammarousCheck --reinstall-languagetool`
- Your OS and Vim version

**Comment by fehrnah** [comment]
OS: Arch linux
vim: neovim 0.2.3

This is exactly the error I got. As you can see the correction provided is "replacement"; but this is not a valid correction (replacement is not even a valid french word).
"replacement" is the label of the suggestions in the XML returned by languagetool.

Using the system languagetool(3.9):

`<?xml version="1.0" encoding="UTF-8"?><matches software="LanguageTool" version="3.9" buildDate="2017-09-26 10:01"><language shortname="fr" name="French"/><error fromy="0" fromx="5" toy="0" tox="14" ruleId="HUNSPELL_NO_SUGGEST_RULE" msg="Faute de frappe possible trouvée" shortmsg="Faute de frappe" replacements="" context="J&apos;ai recemment dormis. " contextoffset="5" offset="5" errorlength="9" category="Faute de frappe possible" categoryid="TYPOS" locqualityissuetype="misspelling"/></matches>`

**Comment by rhysd** [comment]
Oh, I got it. I don't know French so I could not notice the correction was wrong. OK, I now can reproduce this issue. Thank you.

**Comment by rhysd** [comment]
@fehrnah I fixed this issue at 04e3564. Please try out the latest.
